orientdb/src/test/java/org/hibernate/ogm/datastore/orientdb/test/jpa/OrientDbQueryParamTest.java

```diff
@@ -43,11 +38,10 @@
  */
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
 @RunWith(Parameterized.class)
-public class OrientDbQueryParamTest {
+public class OrientDbQueryParamTest extends OgmJpaTestCase {
```

Comment:
em.getFactory() returns null when I run the test

I think it is caused by  @RunWith(Parameterized.class)
It probabaly overrides the  @RunWith( )
on the superclass